client/components/paginate/ZoomToggleButton.tsx

```diff
@@ -1,13 +1,12 @@
 import React from "react";
-import PropTypes from "prop-types";
 
-const ZoomToggleButton = ({ toggleZoom, zoom }) => {
-    return <button onClick={toggleZoom}>{`Turn Zoom ${zoom ? "Off" : "On"}`}</button>;
-};
+interface ZoomToggleButtonProps {
+    toggleZoom: void;
```

Comment:
Should this type be `() => void` rather than just `void` since it's a void function, not a void value?

Reply:
Made change.